src/rest-server/package-lock.json

```diff
@@ -0,0 +1,6315 @@
+{
```

Comment:
Remove package-lock file.

Reply:
> 请先签署CLA然后修复[lint错误](https://travis-ci.org/microsoft/pai/jobs/534746935)。

Does the company write Microsoft

Reply:
yes

Reply:
Please use [yarn](https://yarnpkg.com/) as the package manager.

Reply:
ok